lib/kafka_rest_client.js

```diff
@@ -34,6 +34,9 @@ var supportContentType = {
 var emptyFunction = function EmptyFunction() {
 };
 
+var defaultClusterUrl = 'LOGGING_TOPICS';
```

Comment:
I changed this to 'LOGGING TOPICS' with a space instead of _. This name won't conflict with any potential topics might be created by users, Kafka does not allow space in name :)